### ISSUE TITLE
performance

### ISSUE THREAD

**Issue by vrobot-2** [RSAHDA/home-website#1]
it should be rewritten using github sso, react (next.js? blitz.js?), and express/flask/django crud backend,

**Comment by rishitv26** [comment]
thats a good idea

**Comment by vrobot-2** [comment]
## Stack Idea:
- [Blitz.js](https://github.com/blitz-js/blitz) Frontend - Possibly Login for developer and customer portal (I know [React](https://github.com/facebook/react) and [Next.js](https://github.com/vercel/next.js) I can do this)
- [Express.js](https://github.com/expressjs/express)/[Django](https://github.com/django/django) Backend - I have used both with DB so I can do either with SQLite or MySQL I need help with backends though since I am not that good.
- Zoho - After you have chosen your domain you can use Zoho to make custom domain emails for the organization, free to begin with (ex: virponugoti@viratwiz.com (actually works, don't contact work email without reason))
- MySQL for Express.js or SQLite for Django.

    We could instead rewrite the entire thing from scratch in Django (again) so it looks good. The CSS doesn't look good, might want to improve that. I like starting from scratch so [DaisyUI](https://github.com/saadeghi/daisyui) + [tailwindcss](https://github.com/tailwindlabs/tailwindcss) is good, but for projects like this where we are creating dashboard for creating projects we can be lazy and just make it in bootstrap or chakra. We could also be completely esoteric and use plain minimal CSS (current) and slow authentication with no SSO, but we can do better. (*right?*)

**Comment by rishitv26** [comment]
You are correct, I just didn't have the brain cells at that moment to do better than what we have. Now though, we can DEFINITLY make it better.
